### ISSUE TITLE
Also replace standalone x's and y's

### ISSUE THREAD

**Issue by Jezzamonn** [MathiasPaumgarten/SmartDuplicate-Sublime#3]
I do a lot of game/graphics programming, it's common to have x's and y's as variables, and it'd be nice if they could change to.

One potential regex to match these is `\bx\b`, as `\b` represents a word boundary. If you space out variables that's fine, but not sure how well that plays out if it's directly next to other symbols.

Not sure if there are any false positives here, like reasons you wouldn't want to make that swap.

This is something I might do if I find myself bothered by it enough.

**Comment by roboshoes** [comment]
It's funny you mention that. It's actually something that has bothered me for a long time as well. I don't fully remember what the reason was for not having it. I guess at the time I wasn't too versed in neither Python nor Regex. I've moved away from Sublime to VSCode over the years. I don't actually have the same plugin but I might write it again for VS Code.

In all honesty, I don't know if I'll have time in my calendar adding any additional features quickly since I don't have tests for this and don't even have Sublime any more. 

If you ever get around to doing it, that would be awesome. Thanks for the interest as well!

**Comment by Jezzamonn** [comment]
Yeah, I'm more of filing a bug to mark this as a feature that I might implement rather than an expectation for someone to add it for me. :)

**Comment by roboshoes** [comment]
Ha. that's awesome. I dig it. 👬
